src/cli/commands/script.ts

```diff
@@ -28,15 +48,11 @@ export async function runScript(network: string, script: string, scriptArgs: any
     network,
     args: scriptArgs,
     env: process.env,
-    setTimeout
+    setTimeout,
+    require: vmRequire,
```

Comment:
can you give an example of this being used? Seems like I'm missing some context here that would be good to get

Reply:
Yup, the vm includes nothing by default (not like it's actually a secure sandbox since it's trivial to break out, but it does try at least basic isolation). So we need it to include a line like this: https://github.com/compound-finance/compound-protocol-alpha/pull/34/files#diff-820f964ebee0ceadfb869d7275535674R1. The other problem is that it works fine for `require('fs')` or such, but it fails when you do relatives since it's relative to saddle, not the local file. We add a minor-hack to re-adjust relatives in the scripts.